packages/frontend/src/components/ChartConfigPanel/ChartConfigPanel.styles.ts

```diff
@@ -7,6 +7,9 @@ export const InputWrapper = styled(FormGroup)`
         display: inline-flex;
         gap: 3px;
     }
+    & .bp3-html-select {
+        width: 100%;
+    }
```

Comment:
I think we can remove this when using the `fill` property in the select component